ansible/vagrant/README.md

```diff
@@ -199,6 +199,19 @@ kube-master | SUCCESS => {
 ...
 ```
 
+Example to launch one playbooks:
```

Comment:
Why do we need it? The `deploy-cluster.yml` playbook is run when ``vagrant up`` is executed.